concepts/eventing/performance evaluation/nats-jetstream/results_prod.md

```diff
@@ -109,20 +124,32 @@ kubectl scale statefulset eventing-nats -n kyma-system --replicas 3
   - Redelivered Messages: 0
   - Unprocessed Messages: 0
 
+**Test Results:**
 ```
 -> Total Events Sent      by **Test Sender** : 71,634 (+ 11,113 Failed = 82,747)
 -> Total Events Received  by **Stream**      : 71,634 (i.e. 149,259 - 77,625)
 -> Total Events Processed by **Consumer**    : 71,634 (i.e. 149,259 - 77,625)
 -> Total Events Received  by **Sink**        : 71,637 (Means that 3 events were duplicates)
 ```
 
-![](assets/crash2_1.png "")
+**Test Results Dashboard:**
+
 ![](assets/crash2_2.png "")
+
+**Test Sender Stats:**
+
+![](assets/crash2_1.png "")
+
+**NATS Stream Info:**
+
 ![](assets/crash2_4.png "")
+
+**NATS Consumer Info:**
+
 ![](assets/crash2_3.png "")
 
 ---
----
+
 ## Test Scenario 4: Eventing-controller Pod deleted during test
```

Comment:
Feedback: With the `findings` section it is now **easier** to follow you. Thanks for adding this and very interesting experiments 👍😎